recipes/sirius-ms/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "5.6.4" %}
+{% set commit = "6e51fdee75002bf2b91597b9318b98b674939a9a" %}
+
+package:
+  name: sirius-ms
+  version: {{ version }}
+
+source:
+  - url: https://github.com/boecker-lab/sirius/archive/{{ commit }}.zip
+    sha256: efc618bf61b63ebcba34466fa197b10e1cd1c7ad89586abba87f2a76ab48f166
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - openjdk
+  run:
+    - openjdk
+    - coin-or-cbc  # [not win]
+
+test:
+  requires:
+    - wget  # [not win]
+
+about:
+  home: https://bio.informatik.uni-jena.de/software/sirius/
+  license: AGPL-3.0-only
```

Last thing to do is to update this license line with all the licenses that you have collected. You can use OTHER if the license does not have an identifier.

```suggestion
  license: AGPL-3.0-only AND MIT AND ....
```
Docs page: https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions
Identifiers: https://spdx.org/licenses/

---

Many of them have an identifier but not all of them. So I would use the  `AND OTHER`  as you suggested.